tests/validation/tests/v3_api/test_secrets.py

```diff
@@ -623,12 +623,33 @@ def create_project_client(request):
     namespace["project"] = p
     namespace["c_client"] = c_client
 
+
     def fin():
         client = get_user_client()
         client.delete(namespace["project"])
+
     request.addfinalizer(fin)
 
 
+def rancher_version_check():
+    # Checks if rancher version is greater than v2.6
```

Comment:
This checks if the rancher version is > v2.5*

It might be useful to allow passing the version to compare against as a parameter? 
e.g. `def rancher_version_check(compare_version='v2.5'): `

That way we compare rancher_version vs compare_version instead of only being able to compare against "v2.5"

Reply:
this might not be required as it the comparison is with 'v.5' only